vignettes/summarizeOverlaps.Rmd

```diff
@@ -0,0 +1,328 @@
+---
+title: "Counting reads with `summarizeOverlaps`"
+author:
+- name: Valerie Obenchain
+- name: Rohit Satyam
+  affiliation: "Vignette translation from Sweave to Rmarkdown / HTML"
+clean: false
+date: "`r format(Sys.Date(), '%d %B, %Y')`"
+package: GenomicAlignments
+vignette: >
+  %\VignetteIndexEntry{Counting reads with summarizeOverlaps}
+  %\VignetteEncoding{UTF-8}  
+  %\VignetteDepends{GenomicAlignments, DESeq2, edgeR, rtracklayer}
+  %\VignetteKeywords{sequence, sequencing, alignments}
+  %\VignettePackage{GenomicAlignments}
+  %\VignetteAuthor{Valerie Obenchain}
+  %\VignetteEngine{knitr::rmarkdown}
+bibliography:
+- references.bib
+link-citations: TRUE
+reference-section-title: References
+
+output: 
+    BiocStyle::html_document:
+      toc_float: true
+      fig_caption: yes
+editor_options: 
+  markdown: 
+    wrap: 72
+---
+
+```{r options, echo=FALSE, include=FALSE}
+knitr::opts_chunk$set(collapse = TRUE,eval = T)
+options(width=72)
+options("showHeadLines" = 3)
+options("showTailLines" = 3)
+```
+
+# Introduction
+
+This vignette illustrates how reads mapped to a genome can be counted
+with `summarizeOverlaps`. Different "modes" of counting are provided to
+resolve reads that overlap multiple features. The built-in count modes
+are fashioned after the "Union", "IntersectionStrict", and
+"IntersectionNotEmpty" methods found in the HTSeq package by Simon
+Anders [@Anders2019][@Putri2022].

```

Comment:
This produces `[Putri et al. (2019)](Putri et al. 2022)`. If you put a space in between the two references, they will render as `(Putri et al. 2019) (Putri et al. 2022)`. Please insert a space.